pkg/testutil/testutil_test.go

```diff
@@ -0,0 +1,174 @@
+package testutil
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/gitpod-io/leeway/pkg/leeway"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+)
+
+func TestLoadFromYAML(t *testing.T) {
+	ignores := cmpopts.IgnoreUnexported(
+		leeway.Workspace{},
+		leeway.Component{},
+		leeway.Package{},
+		leeway.WorkspaceProvenance{},
+		leeway.GitInfo{},
+	)
+
+	tests := []struct {
```

Comment:
Nice to have: A simple test that shows that it errors if you give it malformed YAML

Reply:
I've indeed taken the shortcut and built the expectation without the error, i.e. made the assumption that any error would represent test failure. This makes the integration of non-happy path tests harder ...